src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog__content {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, 406px);
```

Comment:
don't hardcode the height of cards. this property can be removed so the catalog can have unlimited number of rows